assets/js/llms-builder.js

```diff
@@ -3224,7 +3224,7 @@ define( 'Collections/Questions',[ 'Models/Question' ], function( model ) {
  * Quiz Schema
  *
  * @since    3.17.6
- * @version  3.24.0
+ * @version  [version]
```

Comment:
this whole file shouldn't be in the PR, it's a compiled one.

Reply:
I've reverted the changes.